### ISSUE TITLE
Comparing baseline benchmarks

### ISSUE THREAD

**Issue by mistakia** [orbitdb/benchmark-runner#26]
To make it easier to compare benchmarks, I was thinking about adding a cli flag that would be used to write the baseline benchmark results to a provided path. Then this file can be loaded to provide a diff between two baseline benchmarks (i.e. comparing a PR to master). 

Let me know if there's a better way to do this and how it could possibly integrate with CircleCI so that perhaps we could automate comparing baseline benchmarks for PRs.

**Comment by aphelionz** [comment]
I think this sounds like a good idea as any. Anything that would get us automated comparative benchmarks like this is welcome.

**Comment by mistakia** [comment]
My understanding of CircleCI is very limited but I was imagining it would run the benchmark in master and provide a globally accessible path. Then it would run the benchmark in that branch and the benchmark runner has access to that previous path and can output a report. 

Not sure if this is possible.

**Comment by aphelionz** [comment]
Yes, I believe it is via build artifacts. If you find it's easier in a different CI system i.e. GH actions that's fine too. I like CircleCI because I like a healthy ecosystem of tools instead of getting further locked-in to GH, but we're kinda there already anyway :P
